.kokoro/presubmit_tests_composer.cfg

```diff
@@ -11,5 +11,13 @@ env_vars: {
 
 env_vars: {
     key: "NOX_SESSION"
-    value: "composer and py27 and not appengine"
+    value: "composer and py36 and not appengine"
+}
+
+# Explicitly allow GPL dependencies to let the apache-airflow package
+# installation continue. See:
+# https://github.com/apache/incubator-airflow/pull/3660
+env_vars: {
+    key: "AIRFLOW_GPL_UNIDECODE"
```

Comment:
Would SLUGIFY_USES_TEXT_UNIDECODE work?

Reply:
Done in https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1688/commits/495e7653c5a139a3f37f33d97843d4de08ae61ed

Added environment variable to Travis, too. Looks like this repo hasn't 100% migrated to Kokoro yet.